rts/motoko-rts/src/bigint.rs

```diff
@@ -246,6 +246,28 @@ unsafe extern "C" fn bigint_of_int64(j: i64) -> SkewedPtr {
     persist_bigint(i)
 }
 
+#[cfg(feature = "ic")]
+#[no_mangle]
+unsafe extern "C" fn bigint_of_float64(j: f64) -> SkewedPtr {
+    if j < 1073741824.0 && j > -1073741825.0 {
+        return SkewedPtr(((j as i32) << 1) as usize);
```

Comment:
The `as` operator is a rounding conversion, not just a type cast, right? Does it round the right and same way as the libtommath code? In particular at .5?

Reply:
The tests have fractional parts, and the various technologies coincide, so I am hopeful (pretty certain) that `as` does the right thing!

Reply:
Checked: https://stackoverflow.com/questions/37506672/convert-float-to-integer-in-rust